classes/Http/Controller/ProfileController.php

```diff
@@ -68,16 +82,10 @@ public function editAction(Request $request)
 
     public function processAction(Request $request)
     {
-        /** @var Authentication $authentication */
-        $authentication = $this->service(Authentication::class);
-
-        $userId = $authentication->userId();
-
-        /** @var Session\Session $session */
-        $session = $this->service('session');
```

Comment:
Since we would otherwise be invoking `$request->getSession()` multiple times, can we maybe just use `$session` here and assign it `$request->getSession()`?

Reply:
See the discussion above.